OrbitClientProtos/preset.proto

```diff
@@ -14,3 +14,8 @@ message PresetInfo {
   string process_full_path = 1;
   map<string, PresetModule> path_to_module = 2;
 }
+
+message PresetFile {
```

Comment:
One thing is unclear to me: is this used for serialization?
If not, why does this need to be a proto? Convenience?
If yes (but it doesn't look like it, or is it for the future?), isn't `file_name` redundant (and worse, not in sync with the name of the file in case the file is renamed)?

Reply:
Ok, sorry, missed the comment.